docs/guides/scxml.md

```diff
@@ -0,0 +1,203 @@
+# SCXML
+
+XState is compliant with the [SCXML (State Chart XML : State Machine Notation for Control Abstraction) specification](https://www.w3.org/TR/scxml/). This page contains details on where our API relates to the SCXML specification.
```

Comment:
New language here to suggest that XState is not strictly an SCXML interpreter (it already has more flexibility and slightly different semantics in places) but a statechart interpreter that _can_ interpret SCXML documents (cc. @Andarist)

```suggestion
XState is compatible with the [SCXML (State Chart XML: State Machine Notation for Control Abstraction) specification](https://www.w3.org/TR/scxml/). This page contains details on where our API relates to the SCXML specification.
```